components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatientService.xml

```diff
@@ -109,6 +109,9 @@
     #set ($data.unauthorizedserver = 'true')
   #elseif ($response.isLoginFailed())
     #set ($data.loginfailed = 'true')    ## one form or another of incorrect username, password or stored token
+    #if ($response.isLoginFailed_UsernameNotCanonical())
+      #set ($data.usernameNotCanonical = 'true')
```

Comment:
Where is this used?

Reply:
https://github.com/phenotips/phenotips/pull/2808/files#diff-6c0b884a881ac3f49d2dcfe3e4d6f737R619

Reply:
Right, thank you.